### PR TITLE
Fix user urls

### PR DESCRIPTION
**Pull request by NLaws**
## Bug fixes
- The `v1/user/<user_uuid>/unlink/<run_uuid>` end point was not working due to unclear path definitions in `summary/urls.py` and `reopt_api/urls.py`
- Once the URLs were fixed, the 204 code forced the `JSONResponse` to be  empty (because that is what is expected for 204). Changed it to 208.
## Changes
-  clarified the `add_user_uuid` view (which has never been used except by `summary/tests/test_summary_and_unlink.py`)
- clarified help text for `Jenkinsfile-undeploy`

### CODE REVIEW

**Comment by NLaws** [comment]
@hdunham can you please prioritize this review? It is causing many errors in production

File: summary/views.py [hunk 1]
```diff
@@ -40,65 +39,53 @@
 from summary.models import UserUnlinkedRuns
 
 
-def add_user_uuid(request):
+def add_user_uuid(request, user_uuid, run_uuid):
```
Comment (hdunham) [inline summary/views.py:42]: request param no longer needed?

Comment (hdunham) [reply summary/views.py:42]: wouldn't you also need to change the arguments being passed wherever this function is called? I'm not seeing that.

**Review by hdunham — APPROVED** [review]

File: summary/views.py (hunk 1 above)
Comment (NLaws) [reply summary/views.py:42]: The `views.py` functions are used by Django as the endpoints and have to implement `request` as the first argument. https://docs.djangoproject.com/en/3.2/topics/http/views/

**Pull request merged**